src/lib/entrypoint.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @fileoverview Site bootstrap code.
```

Comment:
Should this line say something else? I do worry that entrypoint and bootstrap are synonyms so it's still unclear at a glance which comes first...

Reply:
I know 'bundle' is a bit of a noise word, but it is idiomatic in something like webpack to say that bundle is the thing everything has been packed into. Kinda wondering if that would be a better name for this file... I realize I'm bikeshedding but it is confusing at a glance

Reply:
I didn't catch this comment. I've updated it a whole bunch to be clearer about what we do here.